tests/MongoDB.Driver.Core.Tests/Core/Authentication/ScramSha256AuthenticatorTests.cs

```diff
@@ -229,7 +231,19 @@ public void Constructor_should_throw_an_ArgumentNullException_when_credential_is
                    ok: 1}"));
 
             var connection = new MockConnection(__serverId);
-            connection.EnqueueReplyMessage(saslStartReply);
+            var isMasterResult = (BsonDocument)__description.IsMasterResult.Wrapped.Clone();
```

Comment:
The format of this test is the same as the SCRAM-SHA-1 test, so all explanatory comments for that file also apply here.